package.json

```diff
@@ -72,6 +72,7 @@
     "prettier": "^1.18.0",
     "raf": "^3.4.0",
     "raw-loader": "^0.5.1",
+    "react-dnd-test-backend": "^7.4.4",
```

Comment:
I think this will be pulled in via `@theforeman/vendor`, so bumping its version to 1.6.0 should make Travis happy.

Reply:
exactly, I do not know if the test related stuff should go in `@theforeman/vendor` or not

Reply:
No need anymore, it's been bumped in other PR and even packaged (theforeman/foreman-packaging#4142)